OpenOversight/app/models/database.py

```diff
@@ -61,6 +61,24 @@ def to_custom_dict(self):
             "unique_internal_identifier_label": self.unique_internal_identifier_label,
         }
 
+    def total_documented_officers(self):
+        return (
+            db.session.query(Officer).filter(Officer.department_id == self.id).count()
+        )
+
+    def total_documented_assignments(self):
+        return (
+            db.session.query(Assignment.id)
+            .join(Officer, Assignment.officer_id == Officer.id)
+            .filter(Officer.department_id == self.id)
+            .count()
+        )
+
+    def total_documented_incidents(self):
+        return (
+            db.session.query(Incident).filter(Incident.department_id == self.id).count()
+        )
+
```

Comment:
Nice addition! Only concern I have is performance, since for `/browse` we have to do this for every department, so `3 * num departments` queries. Maybe a good time to bring in the query caching you had told me about. How could we enable something like that?

Reply:
Most of this came from inspiration I saw based on @AetherUnbound's work:
https://github.com/OrcaCollective/OpenOversight/blob/main/OpenOversight/app/models.py#L88C1-L95

I don't see why we couldn't implement a fairly straight-forward cache here similar to what they accomplished on their branch.

Reply:
Most of this came from inspiration I saw based on @AetherUnbound's work:
https://github.com/OrcaCollective/OpenOversight/blob/b8bf6394d8f357bd23f0d03adc2b1b573cf5d329/OpenOversight/app/models.py#L88C1-L95

I don't see why we couldn't implement a fairly straight-forward cache here similar to what they accomplished on their branch.

Reply:
https://github.com/lucyparsons/OpenOversight/issues/997

Reply:
Done!